### DISCUSSION TITLE
Icone Off inerte

### DISCUSSION THREAD

**Discussion by Nissa3306** [jmcollin78/versatile_thermostat#699] [Ideas]
Bonjour,
Tout d'abord merci pour ce superbe thermostat et les 2 maj de ces derniers jours.
Une petite suggestion : L'icone Off reste inerte lorsqu'on l'active. Pourrait-on la colorer ou la mettre en gras lorsqu'elle est activée ? Comme la On qui passe au rouge lors de son appui. L'état du thermostat serait ainsi plus clair.
Merci

**Comment by jmcollin78** [comment]
Hello, tu peux me faire voir ce que tu veux dire.

**Comment by Nissa3306** [comment]
Le thermostat fonctionne, l'icone On devient rouge, l'icone Off est grisée
Et là le thermostat est arrêté, l'icone Off devient très légèrement grisée alors que c'est l'état en cours, l'icone On est black
![Off](https://github.com/user-attachments/assets/7f09c84f-54f4-47dc-953c-8bc901c99cb3)
![On](https://github.com/user-attachments/assets/2082886a-973b-4ed3-8079-47a028a4addb)

**Reply by jmcollin78** [reply]
Ah ok, c'est pas pour Versatile Thermostat mais pour la UI Card qui est un autre projet ici: https://github.com/jmcollin78/versatile-thermostat-ui-card

Je ne peux pas transférer inter-projet.

**Reply by Nissa3306** [reply]
Ah Ok, merci et désolé
